_data/chains/eip155-8453.json

```
@@ -1,7 +1,7 @@
 {
   "name": "Base",
   "chain": "ETH",
-  "rpc": ["https://mainnet.base.org/"],
+  "rpc": ["https://mainnet.base.org/", "https://developer-access-mainnet.base.org"],
```

nit: multiple lines and I think we need a slash at the end 
```suggestion
  "rpc": [
      "https://mainnet.base.org/",
       "https://developer-access-mainnet.base.org/"
    ],
```